### ISSUE TITLE
Create a 0.7.1 release

### ISSUE THREAD

**Issue by bkiers** [bkiers/Liqp#63]
☝️

**Comment by mosabua** [comment]
If everything in master is ready to go I can cut a release and get it onto Central. Please confirm that you want to go to 0.7.0 as version as well.

**Comment by bkiers** [comment]
Yes, you can use version 0.7.0 (`master` branch).

Release notes: https://github.com/bkiers/Liqp/releases/tag/0.7.0

Many thanks @mosabua!

**Comment by mosabua** [comment]
Deployment done. https://repository.takari.io/content/repositories/releases/nl/big-o/liqp/0.7.0/

Should be synced to central in a bit. Check in a few hours and let me know if it hasnt made it in case I have to follow up.

**Comment by mosabua** [comment]
It has not yet landed on Central and I am investigating what is going on with the sync http://repo1.maven.org/maven2/nl/big-o/liqp/

Stay tuned

**Comment by mosabua** [comment]
So ...  it turns out that the sync to central is deactivated so I have to rejig the release process. Two options here

1. We forget about 0.7.0 and I fix up the release process and release 0.7.1 via ossrh
2. We delete tags and such in github, turn back to 0.6.0-SNAPSHOT and release 0.7.0 again to central. 

Your choice @bkiers .. just let me know and I will prepare a PR for master or develop (or both) and then run a release off master

**Comment by bkiers** [comment]
Hi @mosabua, let's just go to 0.7.1: https://github.com/bkiers/Liqp/releases/tag/0.7.1

I've updated and tagged `master` for 0.7.1.

Thanks!

**Comment by mosabua** [comment]
Since you tagged in github already I had to rev again. I did all the necessary changes in the POM in master and cut a release of 0.7.2.

It is currently already in OSSRH. 
https://oss.sonatype.org/content/repositories/releases/nl/big-o/liqp/0.7.2/

Release process with the release plugin is documented in the readme file.
New release process automatically revs versions in master and uses just the version as tag.

https://github.com/bkiers/Liqp/tree/0.7.2

Binaries should be on Central shortly. 
http://repo1.maven.org/maven2/nl/big-o/liqp/

Master branch is now at 0.7.3-SNAPSHOT so it is essentially the new development branch. 

I hope that works for you. Let me know if I can help with anything or clarify things. You might want to clean up tags and release in github as well.
